boards/mro/cannode/uavcan_board_identity

```diff
@@ -0,0 +1,17 @@
+# UAVCAN boot loadable Module ID
+set(uavcanblid_sw_version_major 0)
+set(uavcanblid_sw_version_minor 1)
+add_definitions(
+	-DAPP_VERSION_MAJOR=${uavcanblid_sw_version_major}
+	-DAPP_VERSION_MINOR=${uavcanblid_sw_version_minor}
+)
+
+set(uavcanblid_hw_version_major 3)
+set(uavcanblid_hw_version_minor 236)
+set(uavcanblid_name "\"org.ardupilot.mro_f303\"")
```

Comment:
We should update this.

Reply:
Agreed. I think the KitCan version as the F103 chip, so we would need to find a common naming convention?

Reply:
Changed to Pin12.

Reply:
Changed to pin 12